Documentation/helm-values.rst

```diff
@@ -956,6 +956,10 @@
      - Encryption method. Can be either ipsec or wireguard.
      - string
      - ``"ipsec"``
+   * - :spelling:ignore:`encryption.wireguard.persistentKeepalive`
+     - Controls Wireguard PersistentKeepalive option. Set 0 to disable.
```

Comment:
I think there might be a typo here. Do users need to set to `0` or `0s` to disable?

```suggestion
     - Controls Wireguard PersistentKeepalive option. Set to 0s to disable.
```